language/RELEASES.md

```diff
@@ -0,0 +1,72 @@
+# Move Version 1.2
+
+Version 1.2 of Move (released as part of Diem version 1.2) includes several new language features, a new version of the bytecode format, significant improvements to the Move Prover, and numerous bug fixes.
+
+## Highlights
+
+* Move Language Enhancements: This version of Move adds support for three new language features. Each of these is described in more detail in separate change descriptions.
+    * Abilities: a generalization of the existing `resource`/`struct` distinction to enable more fine-grained control over the operations allowed on a record value.
+    * Script Functions: a new visibility modifier that allows a function to be called only from a transaction or another script function.
+    * Friend Visibility: a new visibility that allows a function to be called only by a set of declared friend modules.
+* Version 2 of the Move bytecode format: The bytecode format has been changed to support the new features. The Move VM still reads and processes older versions of the Move bytecode, but new bytecode files will require the new Move VM version.
+* Move Prover: verification speed improvements of 2x and more via new internal architecture.
+
+## VM
+
+This release includes several changes and enhancements:
+
+* Arguments to Move functions are now specified as BCS-serialized values ([#7170](https://github.com/diem/diem/pull/7170)) and the VM also returns serialized values ([#7599](https://github.com/diem/diem/pull/7599)). The VM’s `execute_function` API now returns the serialized return values ([#7671](https://github.com/diem/diem/pull/7671)).
+* The VM’s file format deserializer now supports versioning so that it can seamlessly read multiple versions of Move bytecode files ([#7323](https://github.com/diem/diem/pull/7323)).
+* The VM’s module publishing API now allows republishing an existing module, as long as the updated module is backward compatible with the previous version ([#7143](https://github.com/diem/diem/pull/7143)). This includes a new bytecode verifier check for module updates that introduce cyclic dependencies ([#7234](https://github.com/diem/diem/pull/7234)) and related checks for cyclic dependencies when building and loading the standard library ([#7475](https://github.com/diem/diem/pull/7475)).
+* A new  `InternalGasUnits` type has been introduced to distinguish the unscaled units within the VM from the scaled `GasUnits` type ([#7448](https://github.com/diem/diem/pull/7448)).
+
+**Fixed bugs:**
+
+* Creating a normalized struct type now correctly uses the module handle associated with the StructHandleIndex rather than the module containing the declaration ([#7321](https://github.com/diem/diem/pull/7321)).
```

Comment:
```suggestion
* Creating a normalized struct type now correctly uses the module handle associated with the `StructHandleIndex` rather than the module containing the declaration ([#7321](https://github.com/diem/diem/pull/7321)).
```